ui/app/models/database/role.js

```diff
@@ -130,6 +130,7 @@ export default Model.extend({
   credentialPath: lazyCapabilities(apiPath`${'backend'}/creds/${'id'}`, 'backend', 'id'),
   staticCredentialPath: lazyCapabilities(apiPath`${'backend'}/static-creds/${'id'}`, 'backend', 'id'),
   canGenerateCredentials: alias('credentialPath.canRead'),
```

Comment:
could this be a computed instead and true if  `staticCredentialPath.canRead` OR 
`credentialPath.canRead` is true? that way the HBS template could still be just `@model.canGenerateCredentials`?

Reply:
Welp. I just realized the param for staticCredentialPath.canRead already existed. The naming makes sense but doesn't at first glance. You can get static roles creds and you can generate dynamic roles creds. The naming difference is get vs generate. To keep this backport with minimum changes, I'll remove my additional param and just keep the logic as is in the hbs template but replace with the param that already existed.

Reply:
Good idea. The type matters here. So if type is static then we have to check the static path and if type is dynamic check the dynamic path. We could still do a computed given this logic but I go back and forth on this. I think because this is a backport, and I just noticed (my bad) that the the static param already existed `canGetCredentials` I'll keep the changes to a minimum. The `canGenerateCreds` and `canGetCredentials` are used in another template so to use a computed I would need to amend the logic there as well. Thoughts?

Reply:
Nice catch! I see that now, good call removing!